### PR TITLE
Use the org_id from the user's context

### PR DESCRIPTION
**Pull request by diversemix**
# Description

- **rename JWTUser to UserContext**
- **bump version**

## Proposed version

Please select the option below that is most relevant from the list below. This
will be used to generate the next tag version name during auto-tagging.

- [ ] Skip auto-tagging
- [x] Patch
- [ ] Minor version
- [ ] Major version

Visit the [Semver website](https://semver.org/#summary) to understand the
difference between `MAJOR`, `MINOR`, and `PATCH` versions.

Notes:

- If none of these options are selected, auto-tagging will fail
- Where multiple options are selected, the most senior option ticked will be
  used -- e.g. Major > Minor > Patch
- If you are selecting the version in the list above using the textbox, make
  sure your selected option is marked `[x]` with no spaces in between the
  brackets and the `x`

## Type of change

Please select the option(s) below that are most relevant:

- [ ] Bug fix
- [ ] New feature
- [ ] Breaking change
- [ ] GitHub workflow update
- [ ] Documentation update
- [ ] Remove legacy code
- [ ] Dependency update

## How Has This Been Tested?

Please describe the tests that you added to verify your changes.

## Reviewer Checklist

- [ ] The PR represents a single feature (small drive-by fixes are also ok)
- [ ] The PR includes tests that are sufficient for the level of risk
- [ ] The code is sufficiently commented, particularly in hard-to-understand areas
- [ ] Any required documentation updates have been made
- [ ] Any TODOs added are captured in future tickets
- [ ] No FIXMEs remain

### CODE REVIEW

File: app/model/jwt_user.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 from pydantic import BaseModel
 
 
-class JWTUser(BaseModel):
+class UserContext(BaseModel):
```
Comment (THOR300) [inline app/model/jwt_user.py:6]: Do we need to change the name of the module from `jwt_user.py` as well?

File: tests/integration_tests/analytics/test_get.py [hunk 2]
```diff
@@ -94,28 +94,6 @@ def test_get_analytics_summary_unfccc(
     }
 
 
-def test_get_analytics_summary_other(
```
Comment (THOR300) [inline tests/integration_tests/analytics/test_get.py:97]: Is this just a drive by removal, I can't see the test replaced anywhere?

File: tests/integration_tests/collection/test_all.py [hunk 3]
```diff
@@ -78,29 +78,6 @@ def test_get_all_collections_unfccc(
     assert actual_data[0] == EXPECTED_COLLECTIONS[3]
 
 
-def test_get_all_collections_other(
```
Comment (THOR300) [inline tests/integration_tests/collection/test_all.py:81]: same as the test above

**Review by THOR300 — APPROVED** [review]
Looks nice, just a couple of queries and a merge module to resolve!

File: tests/integration_tests/analytics/test_get.py (hunk 2 above)
Comment (katybaulch) [reply tests/integration_tests/analytics/test_get.py:97]: Yeah it's no longer necessary.

File: tests/integration_tests/collection/test_all.py (hunk 3 above)
Comment (katybaulch) [reply tests/integration_tests/collection/test_all.py:81]: Also no longer necessary :)

File: tests/unit_tests/service/login/test_token_service.py.orig [hunk 4]
```diff
@@ -0,0 +1,106 @@
+from typing import cast
```
[the remark this answers is not in the document]
Comment (katybaulch) [reply tests/unit_tests/service/login/test_token_service.py.orig:1]: Good spot!

File: app/model/jwt_user.py (hunk 1 above)
Comment (katybaulch) [reply app/model/jwt_user.py:6]: I've renamed to 'user' :)

**Pull request merged**